packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1348,6 +1403,57 @@ describe("normalizeAndValidateConfig()", () => {
   });
 
   describe("named environments", () => {
+    it("should warn if we specify an environment but there are no named environments", () => {
+      const rawConfig: RawConfig = {};
+      const { diagnostics } = normalizeAndValidateConfig(
+        rawConfig,
+        undefined,
+        "DEV"
+      );
+      expect(diagnostics.renderErrors()).toMatchInlineSnapshot(`
+        "Processing wrangler configuration:
+        "
+      `);
+      expect(diagnostics.renderWarnings()).toMatchInlineSnapshot(`
+        "Processing wrangler configuration:
```

Comment:
Did we decide this is going to be a warning and not an error? I'm fine with it, just want to be sure that's where we landed.

Reply:
I wasn't quite sure what we agreed, so I took a call. We can iterate if we decide that a warning is not right here.
I could imagine a case where we want to do some wrangling to something like a KV store with no wrangler.toml at all, in which case an error would be a bit harsh?

Reply:
oh hmm that's a good usecase
ok let's stick with a warning for now and keep an eye on it